notebooks/examples/Identify Inspected Buildings.ipynb

```diff
@@ -0,0 +1,287 @@
LRKNwuJxTRgaA72U8fVWU4RQf8fr8m/0YwGITVwuY9Ti1mEjxfXE52tmNwMIqWE0vSAHrtgUfhVu1IJOKwiieGwWm02rs/zOjRPn64Ff/8ve9i6tSp+NlPr8SH779bsKt3CrZf/ssL+Id/+BLOOeccvPjCs5re70faD+CHP/wBpkyZgl/NvDGrsdl4evmlF/DFL34Rn/nMOXj2qfnVAdk+EZFT6xBY9ksMEh+XFbIR7gJhrsfKISO9LYf/DLfPDsVrQevJd6G3NCORiKPPvCetpHzlvgegp4qP+sqAbc4Ot1tb2Lbb+iv+TipEFpaoWP5wtYjk3JM5yCO0kRgE6cjfJE0Waiz+yzGx8V8uWUSHZaDi17VioO105DdBDntEeMX8jaa0kIu3npExSYWIMvXlHaeUr5Kww+XlkjzWkUgkIEsS/E4eZo13tQljH4yG3mR+c5G/gyENCI6zUOHkLDDmseOs15XWr63rO3VGm/wxlAGSnc0bWiWxeKfn0WPQ7XLB79R+QSqlAZcASShfrnQikYDT7cQpsRMHxikVP6a2gFKPw6NaEPYJ8Lt4KKIDsiTC7/eXfE+XegwODsLjcUMRuKzu8FGfrGm+uqqqsJiZccen1pANAAMDAziu35q247jp8J8Qjg5AdJNYf/DxrJDdtG8OrP2H057Hu7asw9cuuxRnn302brpxBuqXfVTULvFI2P7Li8/ji1/8e0yffh7eWvSapvd7a8sefPvb/xdnn3027vjtbXnlY6e08OWX8KUvfQlTp07F/Ln3QbYRlQHrkfIKCHd8jOCKGzFobio7ZMeC7dh79LnhMdHYPBunqK2IxwcRi4UxGI8ikYjjOLE+Y/y0Hn8Xblm7HViBs8PtdmsG25FIBP1WS8XfSYUoCSCGiuQPV1rUJGRn1XpTHCf61bxh2yVxoAlDTczzSEIPwWGd0HHmVgR0fQqc7LOCNmnoGfcl5FccYEgDKEIPj5i9r6xcqqS7dTWobP3abhGKLJb8co3FYpAFDj7ZDlqDXVszY4JB1zNuhnU+srAEaKIvqzt2xCeBs9JgKAOIPEzLkkZsPUWPRTNLwFBCCXotiCENUBzmvOFVq1zXeDwORZbyiv0qRFE16SruciplLRW3SVZ0KvvRqm7NCtddajOMamcSrlUBAbcIl5jsu/YN9V1X2jU8Ho/D6/VCFjgMuIQxy/lT+epaLQp4PJ4xncjLBdm8zYwBdz92drw07Ca+veMFSC4SWw7/OWfkV+vxvw6XjUe8IpZ/8A4+//nP4Zxp0/C7396GFfVLS+p7HgnbLz7/LL7whc/jggsuwJLFizS933ft3IpLL/0qpk2bhj/8791pjui59NrCl3HxxRdj6tQpuP+Pv4dgMVQesj0cQgfeRLDpV4g7NpYdshHuQrB/NTaO6sve0LYALp9txChLYN+ptzPGz+ZDT8MpaVv+KXA2eDwezZ4joVAIVou54u+kQmRhk2aqnG3i84crJYZzYaNpsOIgUq1aa0zgkDmYt0maS+Jgpoma8JiiCAMc/eYJh+1T/b5PXYvCWQxlhMOc3TwqqsoQbEwahJAmHQbKED+VGwh5TQCulsVQBnAW7WHbp/DwatCvHQgE4BQccJZYgUARehj0PZpmFLK0CZw13WAn4EruwI/8d/IxLWMoY0mwTJrO7BJyqzkJ2y4+vyx4VeE1c7+NxWKQBO1ivyLDpeLaGHhlOwLBAEjBiEPuzFLxDvUTmNROuFUzwqqAsFeCR3JAGuq7jkQiFS0NTx2JRAJ+vx+SwEN18nmb04U8IiRR0KyqQZZlWNjsY7KUCK9sx8DAAPpZEgFX8j3Icyew9fDTwwZW2448nxWwG5tnY1fHS3BJp9+3f3vrNZx33rk4//zpmDP7jyX1PI+G7VTO9p+fehLnnz8dF154Id7922JN7/cN61fj7//+73HOOedg/qNzx/xMi15fiK9ccgmmTJmCP9z1W3Csrgog24FQ818QWnULEtLOCYHsQaYBUsON2PDJaAO02TiiW4F4PLkAFQz7sPXIcxnjaMPBBZBF7SfIAmfTzEMBKLy1o1pEEXo4+ic+f3iiZead2ERMQnY+amHC4MX8TNLcMo9+M10TiTMp9/2Jhm2Dw/OpqqI4y2omYTL0DE8aTgOuAAttAjPKtbRQV2ctVCrAnQmiCB1URfsFDkWjMk6vxwOfwsFhoQounbGwBIyGpOFZeR4meihccqfVxVvBkIYMd9R8TctIorR+bb0uM//7TBNNGvKufPHKHFRfcRFZo49oNApZ4BDJM987lwJuASLPla1UHAA8Xg+6ha60UvF2dTdMahdcKouQyiPqk+BTOMi8A4oiIxgMVgVcp45gMAhJ5OGRizvnqXx1LRYy4vE4eI6DeRRsO+w2TcvpT0P2aV+AqCpB5E9hT9craWXkaZDdch/2HXsLingaLt989SWcd965+Lu/+wKe+/OTaGpYrglkZ4PtJx6bh/POOw8XXfRZfPj+O5rd6xaWwPvvvY3p06fj85//HF5b+HLWz/LWm6/j0ku/irq6OvzPHbPgYPoqD9luO0K7n0Fo7e1IOPdMDGQTHyOw9HrIXQ1Yd2B+xjhZe2AeJDcDIOlqv/HQkxk/s639Bc0M0TJh265ZMgSQXISvRdimSQNsFm2yy6tR/YITW4hYxQGklrSdjMIuKnmfY7u1NhzJGcoIM01M+BhkONenZqHnLKs5CRn9zGl4dgtWMGRuFz2GMkK05efqrJVs5k+3C7nVTIIy6fLOiM5X0aEyTk0ivyRxeIGmEIdwXZGGZ4WIJvWwMqYxF2xIQj9mRndK+jxixnLJzJz5JeTJ822AT86dEzxSTsGBgYEBTSZ2oVAIiuAoOvbLIyfNxMpVKh6Px8HLHLrkg2hVt+KQuhMmtQtOlUFITZZcD7gEyLx9qO+6+vKuw+EwZEmEW3IgUmKWuU/RJgUBSFY12Gz9w88eu91WsOlePBFDJBZEKKIiEgum/VkgkAnZI+V3WUCwn2D/iS
```

Comment:
This is probably the most confusing part of the code, especially if you're not great w Pandas. I'd include a comment on what you're doing here

Reply:
Yeah I agree, I added a comment